webapp/src/components/App/Singlestudy/explore/Xpansion/Candidates/CreateCandidateDialog.tsx

```diff
@@ -15,153 +19,118 @@ interface PropType {
 }
 
 function CreateCandidateDialog(props: PropType) {
-  const [t] = useTranslation();
   const { open, links, onClose, onSave } = props;
-  const [candidate, setCandidate] = useState<XpansionCandidate>({
-    name: "",
-    link: "",
-    "annual-cost-per-mw": 0,
-  });
-  const [toggleView, setToggleView] = useState<boolean>(true);
-  const tabLinks = links.map((item) => `${item.area1} - ${item.area2}`);
+  const [t] = useTranslation();
+
+  ////////////////////////////////////////////////////////////////
+  // State
+  ////////////////////////////////////////////////////////////////
+  const [isToggled, setToggle] = useState<boolean>(true);
+
+  ////////////////////////////////////////////////////////////////
+  // Event Handlers
+  ////////////////////////////////////////////////////////////////
+
+  const handleToggle = () => {
+    setToggle(!isToggled);
+  };
 
-  const handleChange = (key: string, value: string | number) => {
-    setCandidate({ ...candidate, [key]: value });
+  const handleSubmit = (data: SubmitHandlerPlus<XpansionCandidate>) => {
+    const { values } = data;
+
+    if (values) {
+      if (isToggled) delete values["max-investment"];
```

Comment:
put clear accolades.
Also delete max-unit and unit-size when isToggled is false